packages/opentelemetry-scope-base/src/context.ts

```diff
@@ -0,0 +1,72 @@
+/*!
+ * Copyright 2020, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/** Map of identifiers to an unknown value used internally to store context */
+type Store = { [identifer: string]: unknown };
+
+/**
+ * Class which stores and manages current context values. All methods which
+ * update context such as get and delete do not modify an existing context,
+ * but create a new one with updated values.
+ */
+export class Context {
+  private _context: Store;
+
+  public static readonly ROOT_CONTEXT = new Context();
+
+  /**
+   * Construct a new context which inherits values from an optional parent context.
+   *
+   * @param context a context from which to inherit values
+   */
+  protected constructor(parentContext: Store = {}) {
+    this._context = Object.assign(Object.create(null), parentContext);
+  }
+
+  /**
+   * Get a value from the context.
+   *
+   * @param key key which identifies a context value
+   */
+  getValue(key: string): unknown {
```

Comment:
Any update here?

Reply:
I just used `getValue` because that's what the spec used. I think it is supposed to make it clear that you are getting a value and not getting a context. If you want me to change it I can but it seems like keeping it more descriptive is a good thing.

Reply:
WDYT?

Reply:
I would be in favor of `getValue` given your point and the fact that the spec explicitly use that name